Felienne/hedy
# [BUG FIX] Parsons attempts no longer overwritten in database

**Description**
In this PR we attempt to fix the issue where Parsons attempts are being overwritten in the database when the same user (or session.id) solves a new parsons problem. It is still unclear to me why this happens as locally the entries in the database are appended fine instead of overwriting the old ones. Similarly we use the same structure for storing new programs, which also works fine.

@rix0rrr Do you maybe have a clue what is going wrong? We use the Dynamo `create()` function to create a new entry in the table, accompanied with an `attemptID`. For if the username is the same the previous entry is being overwritten. In this PR we attempt to solve this by removing the username as an indexKey, but I don't know why this could be an issue.

**Fixes**
No related issue.

**How to test**
Deploy to Alpha, verify that new Parsons attempts by the same user are stored correctly.
